Resources/views/CRUD/base_show_macro.html.twig

```diff
@@ -0,0 +1,36 @@
+{% macro render_groups(admin, object, elements, groups, has_tab, no_padding = false) %}
+    {% if has_tab %}<div class="row">{% endif %}
+
+    {% for code in groups %}
+        {% set show_group = admin.showgroups[code] %}
+
+        <div class="{{ show_group.class }} {{ no_padding ? "nopadding" }}">
+            <div class="{{ show_group.box_class }}">
+                <div class="box-header">
+                    <h4 class="box-title">
+                        {% block show_title %}
+                            {{ admin.trans(show_group.name, {}, show_group.translation_domain) }}
+                        {% endblock %}
+                    </h4>
+                </div>
+                <div class="box-body table-responsive no-padding">
+                    <table class="table">
+                        <tbody>
+                        {% for field_name in show_group.fields %}
+                            {% block show_field %}
+                                <tr class="sonata-ba-view-container">
+                                    {% if elements[field_name] is defined %}
+                                        {{ elements[field_name]|render_view_element(object)}}
+                                    {% endif %}
+                                </tr>
+                            {% endblock %}
+                        {% endfor %}
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+        </div>
+    {% endfor %}
+
+    {% if has_tab %}</div>{% endif %}
+{% endmacro %}
```

Comment:
Please configure your editor to always add a newline at end of file (the red pictogram should disappear)